rdkit/Chem/MolSurf.py

```diff
@@ -19,6 +19,7 @@
 from rdkit.Chem.PeriodicTable import numTable
 from rdkit.Chem import Crippen
 from rdkit.Chem import rdPartialCharges, rdMolDescriptors
+from rdkit.six.moves import xrange
```

Comment:
Oh, that is a nasty one. Importing xrange added it to the _descList. So much for quick and dirty fixes.